contracts/contracts/core/ProviderRegistryStorage.sol

```diff
@@ -37,6 +37,9 @@ abstract contract ProviderRegistryStorage {
     /// @dev Mapping of provider to withdrawal request timestamp
     mapping(address => uint256) public withdrawalRequests;
 
+    /// @dev Mapping from bidder to provider slashed amount
+    mapping(address => uint256) public bidderSlashedAmount;
+    
     /// @dev See https://docs.openzeppelin.com/upgrades-plugins/1.x/writing-upgradeable#storage-gaps
     uint256[48] private __gap;
```

Comment:
reduce gap

Reply:
No need to reduce, bcs only mapping was added: https://forum.openzeppelin.com/t/storage-gaps-in-libraries-and-structs/28171/2

Reply:
Also upgrades shouldn't happen on the main branch! If we go with an upgrade, gotta follow the readme exactly